lambdas/credentialissuerreturn/src/main/java/uk/gov/di/ipv/core/credentialissuerreturn/CredentialIssuerReturnHandler.java

```diff
@@ -35,6 +35,7 @@
     private static final Logger LOGGER =
             LoggerFactory.getLogger(CredentialIssuerReturnHandler.class);
     private static final String CRI_VALIDATE_ENDPOINT = "/journey/cri/validate/";
+    private static final String JOURNEY_ERROR_ENDPOINT = "/journey/error";
```

Comment:
I'm wondering if these endpoints should be in a shared place in the lib somewhere. I think we're using them in a few places now. Not for this PR though.